core/src/smartcontracts/wasm.rs

```diff
@@ -64,11 +71,15 @@ mod import {
     /// Name of the imported function to execute queries
     pub const EXECUTE_QUERY_FN_NAME: &str = "execute_query";
     /// Name of the imported function to query trigger authority
-    pub const QUERY_AUTHORITY_FN_NAME: &str = "query_authority";
+    pub const GET_AUTHORITY_FN_NAME: &str = "get_authority";
     /// Name of the imported function to query event that triggered the smart contract execution
-    pub const QUERY_TRIGGERING_EVENT_FN_NAME: &str = "query_triggering_event";
-    /// Name of the imported function to query operation that is to be verified
-    pub const QUERY_OPERATION_TO_VALIDATE_FN_NAME: &str = "query_operation_to_validate";
+    pub const GET_TRIGGERING_EVENT_FN_NAME: &str = "get_triggering_event";
```

Comment:
I feel like this form of manual namespacing is part of the reason why Rust so heavily encourages modules. I propose:
```rust 
pub mod fn_names {
    QUERY_TRIGGERING_EVENT: &str = "query_triggering_event",
}
```

Another point here is that the convention should perhaps be to take the common name and using the fact that the `mod` is a namespace just downcase the `stringify!` or even better, just `#[allow]` constants to not be uppercase and have equality between the variable and its string value. Maybe even to the point of just using the variable name and construcing the `&str` in place when necessary?